### PR TITLE
Remove check for constants

### PR DESCRIPTION
**Pull request by planemad**
Per https://github.com/mapbox/mapbox-gl-style-spec/issues/654#issuecomment-272411985

cc @mourner

### CODE REVIEW

**Comment by mourner** [comment]
We should probably hold off merging anything to style-spec until we land https://github.com/mapbox/mapbox-gl-js/pull/3946

**Comment by lucaswoj** [comment]
Feel free to merge this once CI is passing @planemad

**Review by jfirebaugh — APPROVED** [review]

**Pull request merged**
